queuelib/__init__.py

```diff
@@ -1,3 +1,5 @@
 from queuelib.queue import FifoDiskQueue, LifoDiskQueue
 from queuelib.pqueue import PriorityQueue
 from queuelib.rrqueue import RoundRobinQueue
+
+__version__ = "1.6.1"
```

Comment:
A nitpick: it should be before imports accordidng to pep8: https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names

Reply:
Didn't know about that, thanks!